ui/components/Layout.tsx

```diff
@@ -254,8 +260,12 @@ export default function Layout({ children }: any) {
 
               <p className="p-4">Connected to {connectData?.connector?.name}</p>
 
-              <ul className="menu bg-base-100 p-2 -m-2 rounded-box">
-                <li key="address">
+              <ul className="menu bg-base-100 p-2 -m-2 rounded-box space-y-2 rounded-box dark:bg-slate-800">
```

Comment:
If the light theme does not have vertical spacing, do not add vertical spacing to the dark theme.
```suggestion
              <ul className="menu bg-base-100 p-2 -m-2 rounded-box rounded-box dark:bg-slate-800">
```